code/src/components/EmptyList.css

```diff
@@ -0,0 +1,29 @@
+.gif-wrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 0px 0px;
+  justify-items: center;
+  align-items: center;
+}
```

Comment:
Grid love at first sight 😊  But you could remove `grid-template-columns`, `grid-gap` and `align-items`.